cleanlab_studio/internal/tlm/validation.py

```diff
@@ -0,0 +1,178 @@
+import os
+from typing import Union, Sequence, Any
+from cleanlab_studio.errors import ValidationError
+from cleanlab_studio.internal.constants import (
+    _VALID_TLM_MODELS,
+    TLM_MAX_TOKEN_RANGE,
+    TLM_NUM_CANDIDATE_RESPONSES_RANGE,
+    TLM_NUM_CONSISTENCY_SAMPLES_RANGE,
+)
+
+
+SKIP_VALIDATE_TLM_OPTIONS = os.environ.get("CLEANLAB_STUDIO_SKIP_VALIDATE_TLM_OPTIONS", False)
```

Comment:
nit: could be more explicit that this is a boolean flag. the env var will be read as a string and could lead to weird behavior "false" evaluating to True because non-empty strings are truthy. 

one potential option:
`SKIP_VALIDATE_TLM_OPTIONS: bool = os.environ.get("CLEANLAB_STUDIO_SKIP_VALIDATE_TLM_OPTIONS", "false").lower() == "true"`